src/components/Restaurants/RestaurantReview.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { FaSearch } from 'react-icons/fa'; 
+import './RestaurantList.css';
+
+const SearchBar = () => (
+  <header className="header">
+    <div className="searchContainer">
+      <input
+        type="text"
+        placeholder="Restaurants"
+        className="searchInput"
```

Comment:
searchInput and locationInput should have the same padding, currently they are different.